modules/apps/roles/roles-admin-impl/src/main/java/com/liferay/roles/admin/internal/search/spi/model/query/contributor/RoleModelPreFilterContributor.java

```diff
@@ -56,8 +56,9 @@ private void _filterByType(
 		long[] types = GetterUtil.getLongValues(
 			searchContext.getAttribute("types"),
 			new long[] {
-				RoleConstants.TYPE_DEPOT, RoleConstants.TYPE_ORGANIZATION,
-				RoleConstants.TYPE_REGULAR, RoleConstants.TYPE_SITE
+				RoleConstants.TYPE_ACCOUNT, RoleConstants.TYPE_DEPOT,
```

Comment:
no, in this way we are always returning also the account roles by default and it's not the desiderate